drivers/adt7310/Kconfig

```diff
@@ -12,3 +12,8 @@ config MODULE_ADT7310
     select MODULE_PERIPH_SPI
     help
         Driver for the Analog Devices ADT7310 temperature sensor.
+
+config HAVE_ADT7310
+    bool
```

Comment:
Missing selecting the module?

Reply:
There is no saul thus nothing can select it.  This is in preparation for the describe my board that @chrysn and I may start working on at some point...

Reply:
I see